android/src/main/java/io/mosip/tuvali/openid4vpble/exception/exception/ErrorCode.kt

```diff
@@ -0,0 +1,27 @@
+package io.mosip.tuvali.openid4vpble.exception.exception
+
+// Error Code format
+// <Component(2)+Role(1)>(3char)_<Stage>(3char)_<Number>(3char) Eg: TVW-CON-001
+// Stage --> CON(Connection) | KEX(Key Exchange) | ENC(Encryption) | TRA(Transfer) | REP(Report) | DEC(Decryption)
+// ROLE --> TVW(Tuvali+Wallet) | TVV(Tuvali+Verifier)
+// UNK --> If role or stage is not known
+
+enum class ErrorCode(val code: String) {
+  UnknownException("UNK_UNK_001"),
+
+  WalletStateHandlerException("TVW_UNK_001"),
+  WalletTransferHandlerException("TVW_UNK_002"),
+
+  VerifierStateHandlerException("TVV_UNK_001"),
+  VerifierTransferHandlerException("TVV_UNK_002"),
+
+  MTUNegotiationException("TVW_CON_001"),
+  //TODO: Create specific error codes for the below exception
```

Comment:
What is this TODO for?

Reply:
We don't have specific cases for Transfer Failed Exceptions yet. Verifier doesn't send details on why transfer failed and when it happens. Once we implement that logic we can create specific exceptions.